### ISSUE TITLE
Incompatible with Ruby 2.7

### ISSUE THREAD

**Issue by Cysioland** [gael-ian/vagrant-bindfs#92]
When installing it I get an error:

```
Vagrant failed to properly resolve required dependencies. These
errors can commonly be caused by misconfigured plugin installations
or transient network issues. The reported error is:

vagrant-bindfs requires Ruby version ~> 2.2, < 2.7. The current ruby version is 2.7.0.0.
```

**Comment by gael-ian** [comment]
Hi @Cysioland 

Vagrant itself is not compatible with Ruby 2.7.0 (yet), as stated in [the gem specs](https://github.com/hashicorp/vagrant/blob/master/vagrant.gemspec#L15) and confirmed by a core team member in [a recent issue](https://github.com/hashicorp/vagrant/issues/11290#issuecomment-571360711).

I can relax the constraint in vagrant-bindfs but I don't think it will be of any help. You might just encounter different bugs.

**Comment by gael-ian** [comment]
There's nothing in the `vagrant-bindfs` codebase that's wrong with Ruby 2.7.0 but Vagrant validates plugin's dependencies requirements on install and hence blocks it if a constraint fails.

The 1.1.6 release have a simpler constraint on Ruby version and _can_ be installed as a Vagrant plugin even if your system runs Ruby 2.7.0. This will not fix any other Vagrant's Ruby 2.7.0 incompatibility but it's all I can do.
